docs/layouts/shortcodes/troubleshoot.md

```diff
@@ -0,0 +1 @@
+For generic information regarding how to troubleshoot a project , see [Troubleshoot development](/development/troubleshoot.md).
```

Comment:
I wouldn't use the word "generic" when not necessary. It's not really very specific.

I'd also prefer not to have links as titles rather than just parts of the sentence. I feel like it's confusing.

So I'd suggestion something along these lines (but really feel free to adapt it -- at least fix the extra space):
```suggestion
For more general information, see how to [troubleshoot development](/development/troubleshoot.md).
```